x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/percentile.tsx

```diff
@@ -89,7 +89,11 @@ export const percentileOperation: OperationDefinition<
   shiftable: true,
   windowable: true,
   getPossibleOperationForField: ({ aggregationRestrictions, aggregatable, type: fieldType }) => {
-    if (supportedFieldTypes.includes(fieldType) && aggregatable && !aggregationRestrictions) {
+    if (
+      supportedFieldTypes.includes(fieldType) &&
+      aggregatable &&
+      (!aggregationRestrictions || aggregationRestrictions.percentiles)
```

Comment:
It's a slightly weird API - if `aggregationRestrictions` exists, it means not all aggs can be applied. If `aggregationRestrictions.percentiles` is set, it means it can be applied.